pkg/templates/livenessport/template.go

```diff
@@ -0,0 +1,57 @@
+package livenessport
+
+import (
+	"fmt"
+
+	"golang.stackrox.io/kube-linter/pkg/check"
+	"golang.stackrox.io/kube-linter/pkg/config"
+	"golang.stackrox.io/kube-linter/pkg/diagnostic"
+	"golang.stackrox.io/kube-linter/pkg/objectkinds"
+	"golang.stackrox.io/kube-linter/pkg/templates"
+	"golang.stackrox.io/kube-linter/pkg/templates/livenessport/internal/params"
+	"golang.stackrox.io/kube-linter/pkg/templates/util"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const templateKey = "liveness-http-port"
+
+var sentinal = struct{}{}
+
+func init() {
+	templates.Register(check.Template{
+		HumanName:   "Liveness Port Not Open",
+		Key:         templateKey,
+		Description: "Flag containers have a http liveness prop with a port they didn't open.",
```

Comment:
Something like this will be better
```suggestion
		Description: "Flag containers with an HTTP liveness probe to not exposed port.",
```